saleor/graphql/checkout/mutations/checkout_shipping_method_update.py

```diff
@@ -174,13 +174,19 @@ def perform_on_shipping_method(
 
         delete_external_shipping_id(checkout=checkout)
         checkout.shipping_method = shipping_method
+        invalidate_prices_updated_fields = invalidate_checkout_prices(
+            checkout_info, lines, manager, info.context.discounts, save=False
+        )
         checkout.save(
-            update_fields=["private_metadata", "shipping_method", "last_change"]
+            update_fields=[
+                "private_metadata",
+                "shipping_method",
+                "price_expiration",
```

Comment:
Same here, `"price_expiration"` is already in `invalidate_prices_updated_fields`